misc/efi-stub/README.rst

```diff
@@ -0,0 +1,120 @@
+:orphan:
+
+ACRN v2.5 EFI-Stub
+##################
+
+Introduction
+************
+
+``ACRN v2.5 EFI-Stub`` is an EFI application to support booting ACRN Hypervisor on
+UEFI systems with Secure Boot. ACRN has supported `Secure Boot With GRUB
+<https://projectacrn.github.io/latest/tutorials/acrn-secure-boot-with-grub.html>`_.
+It relies on the GRUB multiboot2 module by default. However, on certain platform
+the GRUB multiboot2 is intentionally disabled when Secure Boot is enabled due
+to the `CVE-2015-5281 <https://www.cvedetails.com/cve/CVE-2015-5281/>`_.
+
+As an alternative booting method, ``ACRN v2.5 EFI-Stub`` supports to boot ACRN HV
+on UEFI systems without using GRUB. Although it is based on the legacy EFI-Stub
+which was obsoleted in ACRN v2.3, the new EFI-Stub can boot ACRN HV in the direct
+mode rather than the former de-privilege mode.
+
+In order to boot ACRN HV with the new EFI-Stub, you need to create a container blob
+which contains HV image and Service VM kernel image (and optionally pre-launched
+VM kernel image and ACPI table). That blob file is supposed to be stitched to the
+EFI-Stub to form single EFI application (acrn.efi). The overall boot flow is as below.
+
+UEFI BIOS/shim -> acrn.efi -> ACRN HV -> Service VM (pre-launched VM in parallel)
+
+1. UEFI BIOS/shim verifies & loads a signed acrn.efi
+2. acrn.efi unpacks ACRN HV image and VM kernels from a stitched container blob
+3. acrn.efi loads ACRN HV to memory
+4. acrn.efi prepares MBI to stores Service VM & pre-launched VM kernel info
+5. acrn.efi hands over control to ACRN HV with MBI
+6. ACRN HV boots Service VM (and pre-launched VM in parallel)
+
+As the container blob format, ``ACRN v2.5 EFI-Stub`` uses the `Slim Bootloader
+Container Boot Image <https://slimbootloader.github.io/how-tos/create-container-boot-image.html>`_.
+
+Verified Configurations
+***********************
+- ACRN Hypervisor Release Version 2.5
+- hybrid_rt scenario
+- TGL-U RVP board
+- CONFIG_MULTIBOOT2=y (as default)
+- CONFIG_RELOC=y (as default)
+
+
+Building
+********
+
+Build Dependencies
+==================
+
+- Build Tools and Dependencies described in the `Getting Started Guide <https://projectacrn.github.io/latest/getting-started/building-from-source.html>`_
+- gnu-efi package
```

Comment:
Can you add the `apt` command: `sudo apt install gnu-efi`

Reply:
Is that all that's missing compared to the list of build tools and dependencies for ACRN?